hail/python/hail/plot/plots.py

```diff
@@ -146,6 +147,159 @@ def cumulative_histogram(data, range=None, bins=50, legend=None, title=None, nor
     return p
 
 
+@typecheck(x=expr_numeric, y=expr_numeric,
+           x_range=nullable(sized_tupleof(numeric, numeric)),
+           y_range=nullable(sized_tupleof(numeric, numeric)),
+           bins=int, x_bins=nullable(int), y_bins=nullable(int),
+           plot_title=nullable(str), plot_width=int, plot_height=int,
+           font_size=str, colors=sequenceof(str))
+def histogram_2d(x, y, x_range=None, y_range=None, bins=40, x_bins=None, y_bins=None,
```

Comment:
numpy and matplotlib don't have the `_`. I'd remove it for consistency
https://docs.scipy.org/doc/numpy/reference/generated/numpy.histogram2d.html
https://matplotlib.org/api/_as_gen/matplotlib.pyplot.hist2d.html